packages/env/README.md

```diff
@@ -297,6 +297,24 @@ Positionals:
 
 ### `wp-env run [container] [command]`
 
+The run command can be used to open shell sessions or invoke WP-CLI commands.
+
+<div class="callout callout-alert">
```

Comment:
What does this `<div>` do? I didn't notice anything in the rich diff. Maybe it will only apply after merging?

Reply:
That style class, as I understand it, will format the div as an alert callout. I'm new to the WP doc standards, but I found this information about marking callouts for gutenberg docs: https://developer.wordpress.org/block-editor/contributors/documentation/#callout-notices. So I assume like you it will be applied after merging.

Reply:
As for the json discrepancy, I'm not sure how that happened. I'm guessing the auto-format of Markdown doc in vscode. But all of the json examples in the doc seem to have the same spacing.